packages/core/src/artifacts/ArtifactMigrator.ts

```diff
@@ -1,157 +0,0 @@
-import child_process = require('child_process');
```

Comment:
This is a breaking change, will mean that `SfpowerscriptsArtifact__c` will no longer be migrated to `SfpowerscriptsArtifact2__c`